.travis.yml

```diff
@@ -40,12 +42,14 @@ before_script:
   - psql -c "DROP DATABASE IF EXISTS test_seeddb;" -U postgres
   - mv config/settings/test_local_untracked.py config/settings/local_untracked.py
   - psql -c "CREATE DATABASE seeddb;" -U postgres
+  - psql -d seeddb -c "CREATE EXTENSION postgis;" -U postgres
 env:
   global:
     - DOCKER_COMPOSE_VERSION=1.16.0
     - DJANGO_SETTINGS_MODULE=config.settings.travis
     - DISPLAY=:99.0
     - COVERALLS_REPO_TOKEN=y8UqJm8Bri5ZP8hr3YZM3guBaUKpsfoCv
+    - secure: HtOAAxPiA2NnSxXTBsmmXlFURH6WxnXXeaJLkjKc8cPWMsPX2QSouggtFYCPJxikrvN6bOYvDii67v2++E0qpuPO28LtywaLWu7AdDYxJ+rp4ysVjdi2Pp1bh723SZj2Xi3OMOLPz/kekT41DLKtoEzmBUEFEBIuJ27oU/phNGU=
```

Comment:
This is my encrypted API key. Since VCRpy is being used and cassettes were saved to GitHub (after filtering out the key in the URL), Travis builds won't actually trigger API requests.

Notably, when/if the cassettes are replaced in this repo, the encrypted key should get replaced with whatever key was used to build the cassettes.